Site Factory documentation update (#1025)
ezsystems/developer-documentation: docs/guide/site_factory.md

    - multisite configuration
    
    After [clean installation](../getting_started/install_ez_enterprise.md) the Site Factory will be disabled.

>     If you do not work on clean installation, follow [Upgrading eZ Platform to v3](../updating/upgrading_to_v3.md#site-factory).

Reviewer: ```suggestion
    If you are not working on a clean installation, follow [Upgrading eZ Platform to v3](../updating/upgrading_to_v3.md#site-factory).
```

            example_site_factory_group_2:
    ```
    

>     Uncomment the SiteAccess matcher by removing the comment from `EzSystems\EzPlatformSiteFactory\SiteAccessMatcher` matcher under:

Reviewer: ```suggestion
    Uncomment the SiteAccess matcher by removing the comment from `EzSystems\EzPlatformSiteFactory\SiteAccessMatcher` under:
```

    ```
    
    You can check the results of your work in the Back Office by going to the **Site** tab.

>     There, you should be able to add a new site and choose a design for it.   

Reviewer 2: those empty spaces at the end are not needed

    http://test.example.com:8080/
    ```
    
    ![Site Factory enabled](img/site_factory_site_list.png "Site Factory enabled")
    
    ### Define site directory
        
     You can adjust the place where the directory of the new site will be created.

>      By default the Location for the site driectories is defined in ` src/bundle/Resources/config/default_settings.yaml`:

Reviewer 2: ```suggestion
     By default the Location for the site driectories is defined in bundle configuration `src/bundle/Resources/config/default_settings.yaml`:
```

>      By default the Location for the site driectories is defined in bundle configuration `src/bundle/Resources/config/default_settings.yaml`:

Reviewer: ```suggestion
     By default the Location for the site directories is defined in bundle configuration `src/bundle/Resources/config/default_settings.yaml`:
```

     
     ``` yaml
     parameters:
         ezsettings.default.site_factory.sites_location_id: 2
     ```
    
    To change it to e.g. eZ Platform, go to `config/packages/ezplatform_site_factory.yaml`, and add the following parameter:
    
    ``` yaml
    parameters:
        ezsettings.default.site_factory.sites_location_id: 42
    ```
    
    Now, all new directories will be created under eZ Platform.
    
    ### Provide access
    
    The Site Factory is set up, now you can provide sufficient permissions to the Users.
    
    Set the below Policies to allow Users to:
    
    - `site/view` - enter the Site Factory interface
    - `site/create` - create sites
    - `site/edit` - edit sites

>     - `site/change_status` - change status of the sites to `Live` or `Offline`

Reviewer 2: change status of the Public Accesses not sites

    You can disable Site Factory to boost Config Resolver performance.
    Keep in mind that with disabled Site Factory you will not be able to add new sites or use existing ones.
    
    1\. In `config/packages/ezplatform_site_factory.yaml` change enabled to `false`.

    2\. In `config/packages/ezplatform.yaml` comment the `ezplatform.siteaccess.match: '@EzSystems\EzPlatformSiteFactory\SiteAccessMatcher': ~` if it is uncommented.

    3\. Remove separate connection to database in `config/packages/doctrine.yaml`.
    
    ``` yaml
    doctrine:
        dbal:
            connections:
                ...

>                 # This connection is dedicated for SiteFactory to avoid known issues, should be the same as default

Reviewer 3: ```suggestion
                # This connection is dedicated for SiteFactory to avoid known issues
```
